src/ts/components/extensions/richtexteditor/fragments/RichTextEditor.tsx

```diff
@@ -203,14 +226,66 @@ const RichTextEditor = ({
     };
     // Construct the toolbar. NB: Can't be updated after the editor is created.
     let mantineToolbar = undefined;
+
     // If any extensions are specified, load them. NB: Can't be changed after the editor is created.
-    const mantineExtensions = extensions.map((ext) => {
-        if (typeof ext === 'string') {
-            return extensionMap[ext];
-        }
-        const name = Object.keys(ext)[0];
-        return extensionMap[name].configure(ext[name]);
-    });
+    const mantineExtensions = extensions
+        .map((ext) => {
+            // Case 1: String extension
+            if (typeof ext === 'string') {
+                // Special handling for CodeBlockLowlight (since can't pass lowlight from Python app)
+                if (ext === 'CodeBlockLowlight') {
+                    return extensionMap.CodeBlockLowlight.configure({
+                        lowlight,
+                    });
+                }
+
+                // Validate and return other string extensions
+                if (!(ext in extensionMap)) {
+                    throw new Error(`Unknown extension: "${ext}"`);
+                    return null;
```

Comment:
You won't get here - after throwing the error, execution stops. Seems fine to me since that would have been the effect before this PR (just remove the unnecessary `return null`), but if you'd prefer this to just make a console error and keep going (dropping the unknown extension by returning `null`) then use `console.error` rather than `throw new Error(...)`

Reply:
I started with `console.warn`, but then changed to throwing an error and forgot to remove the `return null`.  I'll fix that.  I think it's better to use the error because it shows up in the Dash dev tools.  I'm probably not the only one who forgets to  check the console when debugging.